docs/source/getting-started/quickstart.rs

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env -S cargo +nightly -Zscript
+```cargo
+# init
+[dependencies]
+opendp = { version = "0.9.2", features = ["contrib", "honest-but-curious"] }
+# /init
+```
+
+// demo
+use opendp::{
+    domains::AtomDomain,
+    measurements::then_base_laplace,
+    metrics::AbsoluteDistance,
+};
+
+fn main() {
+    let space = (AtomDomain::default(), AbsoluteDistance::default());
+    let base_laplace = space >> then_base_laplace(1.0, None);
+    let dp_value = base_laplace.expect("unexpected error").invoke(&123.0);
+    println!("DP value: {}", dp_value.expect("unexpected error"));
+}
```

Comment:
```suggestion
fn main() -> Fallible<()> {
    let space = (AtomDomain::default(), AbsoluteDistance::default());
    let laplace_mechanism = (space >> then_laplace(1.0, None))?;
    let dp_value = laplace_mechanism.invoke(&123.0)?;
    println!("DP value: {}", dp_value);
    Ok(())
}
```

A bit more rust-y. I think you'll also need to use `opendp::error::Fallible`.